client/source/components/shared/form/FormInput.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Really like how you set this up for reuse!

Reply:
Glad to help, hope you find it useful!